packages/neos-ui-redux-store/src/CR/Nodes/selectors.ts

```diff
@@ -66,6 +66,67 @@ export const makeGetDocumentNodes = (nodeTypesRegistry: NodeTypesRegistry) => cr
     }
 );
 
+export const makeGetCollapsableDocumentNodes = (nodeTypesRegistry: NodeTypesRegistry) => createSelector(
```

Comment:
Adjust the wording to `collapsible` everywhere

Reply:
My bad, fixed